juniper_official/routing/collect-isis-lsp-retranx-v1.rule

```diff
@@ -0,0 +1,72 @@
+/*
+ * Collects ISIS interface data for LSP-retransmit till interface-id for release <23.2R2
+ * REQ: IGP RCA- https://paragon-automation.atlassian.net/browse/REQ-1349
+ */
+ healthbot {
+    topic routing.isis {
+        rule collect-isis-lsp-retranx-v1 {
```

Comment:
Why v1 and v2?

Reply:
Please add the details of naming as comments to the rules

Reply:
Added the information in comments- [commit](https://github.com/Juniper/healthbot-rules/pull/974/commits/aa02a0b8ed27241b234f169c72c6f5e9f8145818)